lib/builder/factory.js

```diff
@@ -74,11 +74,11 @@ factory.getExecutablePath = function (platform) {
     if (platform === 'aix') {
         ret = '/usr/sbin/ping';
     } else if (factory.isLinux(platform)) {
-        ret = '/bin/ping';
+        ret = v6 ? '/sbin/ping6' : '/bin/ping';
     } else if (factory.isWindow(platform)) {
         ret = process.env.SystemRoot + '/system32/ping.exe';
```

Comment:
@mondwan click "Files changed" at the top, then hover over the line number and click when you want to add a line-specific comment. :)